src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -783,8 +783,8 @@ private void CheckStake(Block block, ChainedBlock chainTip)
             }
 
             this.logger.LogTrace("Worker #{0} found the kernel.", workersResult.KernelFoundIndex);
-	        //Get reward for newly created block
-	        long reward = fees + this.posConsensusValidator.GetProofOfStakeReward(chainTip.Height+1);
+            // Get reward for newly created block.
+            long reward = fees + this.posConsensusValidator.GetProofOfStakeReward(chainTip.Height+1);
```

Comment:
`chainTip.Height + 1`

As mentioned before, Ctrl+K+D in visual studio should do this for you